Woof/WoofTests/Model/Owner/CurrentOwner+TestData.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+extension Owner.CurrentOwner {
+    /// Interface with test values for unit tests.
+    enum TestData {
+        /// The instance of owner using in unit test as a default instance.
+        static let defaultOwner = Owner(
```

Comment:
nit:

```suggestion
        static let kateAnderson = Owner(
```

to reflect semantics?